vitrina/uapi/views.py

```diff
@@ -111,6 +112,7 @@ def setup(self, request, *args, **kwargs) -> None:
             organization=self.organization,
             is_archived=False
         )
+        self.request_history = RequestHistory.objects.filter(agent=self.object)
```

Comment:
```suggestion
        self.request_history = self.object.requesthistory_set.all()
```

Just add `.prefetch_related("requesthistory")` to `Agent` query

Reply:
Added